scala/src/main/scala/smile/clustering/k-modes/KModes.scala

```diff
@@ -6,6 +6,11 @@ import smile.clustering.distances.Binary_Distance
 
 class KModes(data: Array[(Int, Array[Int])], k: Int, epsilon: Double, jmax: Int, metric: Binary_Distance) extends ScalaClusteringTypes
```

Comment:
How does k-modes compared to k-medoids? We implement `CLARANS`, which is an efficient algorithm for k-medoids.